preflight-packages/examples.jetstack.io/pods_basic/pods.rego

```diff
@@ -3,67 +3,74 @@ package pods
 # See https://github.com/jetstack/preflight/blob/master/docs/datagatherers/k8s_pods.md for more details
 import input["k8s/pods"] as pods
 
-# Rule 'container_cpu_limit'
-default preflight_container_cpu_limit = false
-preflight_container_cpu_limit {
-        count(containers_without_cpu_limits) == 0
-        count(init_containers_without_cpu_limits) == 0
-}
-format_container(pod, container) = name {
-        name := {"namespace": pod.metadata.namespace,
-                 "pod": pod.metadata.name,
-                 "container": container.name}
-}
-cpu_limit_unset(container) {
-        not container.resources.limits.cpu
+format_container(pod, container) = data {
```

Comment:
I don't think this is used any more.

Reply:
Ahh yes, I forgot to clean that out. Fixed in db036d556961bafbb5e1393f4b7991d9f048bc80